### PR TITLE
Add kotlin compiler to docker image chip-build-android

### PR DESCRIPTION
**Pull request by yufengwangca**
We need kotlin compiler in  docker image chip-build-android to build TLV lib

### CODE REVIEW

**Comment by yufengwangca** [comment]
I saw the following message, it looks the maximum kotlin version we support is 1.5.1
 
`2023-03-01 16:52:07 WARNING e: /home/yufengw/.gradle/caches/transforms-3/05ad40720cedd0f6d247cc74067712bf/transformed/jetified-CHIPController.jar!/META-INF/main.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.5.1.`

File: integrations/docker/images/chip-build-vscode/Dockerfile [hunk 1]
```diff
@@ -61,17 +61,6 @@ RUN set -x \
     && chmod -R a+w /opt/android/sdk/licenses \
     && : # last line
```
Comment (andy31415) [inline integrations/docker/images/chip-build-vscode/Dockerfile:63]: why did you remove kotlin from vscode?

I would rather expect

```
COPY --from=android /usr/lib/kotlinc /usr/lib/kotlinc
# ...
ENV PATH $PATH:/usr/lib/kotlinc/bin
```

**Review by andy31415 — APPROVED** [review]
